json/test/targeting-only/positive/basic-json-ops.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "../../../targeting.json",
+  "targeting": {
+    "*" : [
```

Comment:
We don't need a ton of tests here, just 1 positive and 1 negative (above) to makes sure `targeting` can be used in isolation. The rest is covered with the definitions tests.